backend/.env.example

```
@@ -1,99 +1,12 @@
-# ------------------------------------
-# Server
-# ------------------------------------
 PORT=7860
-# NODE_ENV options: development | production | test
-NODE_ENV=development
-
-# ------------------------------------
-# Database (MySQL / TiDB)
-# ------------------------------------
-DB_HOST=localhost
+DB_HOST=127.0.0.1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_HOST key should go before the PORT key

(UnorderedKey)